Motorcycle.js

```diff
@@ -1,6 +1,20 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle() {
```

Comment:
You'll want to define some parameters in here, or else nothing will write to the object properly.
```
function Motorcycle(make, model, year, color, seats, passengers) {
	Car.call(this, make, model, year, color, seats, passengers);
```